gradle/spotbugs-exclude.xml

```diff
@@ -105,6 +127,19 @@ For a detailed description of spotbugs bug categories, see https://spotbugs.read
         <Bug pattern="NP_NONNULL_PARAM_VIOLATION"/>
     </Match>
 
+    <!-- false positive, see:
+       https://github.com/spotbugs/spotbugs/issues/600
+       https://github.com/spotbugs/spotbugs/issues/1791
```

Comment:
Looks like https://github.com/spotbugs/spotbugs/commit/720af6cdd592d2e49d3a472465eed9ad361ebd27 would help, but it's not available in a released version yet.